GitHubProjectCards.ps1

```diff
@@ -0,0 +1,515 @@
+        }
+
+        return Invoke-GHRestMethod @params
+    }
+}
+
+function Move-GitHubProjectCard
+{
+<#
+    .DESCRIPTION
+        Move a GitHub Project Card.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER Card
+        ID of the card to move.
+
+    .PARAMETER Top
+        Moves the card to the top of the column.
+
+    .PARAMETER Bottom
+        Moves the card to the bottom of the column.
+
+    .PARAMETER After
+        Moves the card to the position after the card ID specified.
+
+    .PARAMETER ColumnId
+        The id of a column in the same project to move the card to.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no command line status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Move-GitHubProjectCard -Card 999999 -Top
+
+        Moves the project card with id 999999 to the top of the column.
+
+    .EXAMPLE
+        Move-GitHubProjectCard -Card 999999 -Bottom
+
+        Moves the project card with id 999999 to the bottom of the column.
+
+    .EXAMPLE
+        Move-GitHubProjectCard -Column 999999 -After 888888
+
+        Moves the project card with id 999999 to the position after the card id 888888.
+        Within the same column.
+
+    .EXAMPLE
+        Move-GitHubProjectCard -Column 999999 -After 888888 -ColumnId 123456
+
+        Moves the project card with id 999999 to the position after the card id 888888, in
+        the column with id 123456.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess)]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSShouldProcess", "", Justification = "Methods called within here make use of PSShouldProcess, and the switch is passed on to them inherently.")]
+    param(
+        [Parameter(Mandatory)]
+        [int64] $Card,
+
+        [switch] $Top,
+
+        [switch] $Bottom,
+
+        [int64] $After,
+
+        [int64] $ColumnId,
+
+        [string] $AccessToken,
+
+        [switch] $NoStatus
+    )
+
+    Write-InvocationLog
+
+    $telemetryProperties = @{}
+
+    $uriFragment = "/projects/columns/cards/$Card/moves"
+    $apiDescription = "Updating card $Card"
+
+    $paramsCount = 0
+    foreach ($key in $PSBoundParameters.Keys)
+    {
+        if ($key -in ('Top', 'Bottom', 'After'))
+        {
+            if($PSBoundParameters[$key] -ne $false)
+            {
+                $paramsCount ++
+            }
+        }
+    }
+
+    if($paramsCount -ne 1)
```

Comment:
You can drop the whole $paramsCount section above, and just replace this if like so:

```suggestion
    if (-not ($Top -xor $Bottom -xor ($After -gt 0)))
```

For more info on xor, [see here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_logical_operators).